src/som/interpreter/actors/Actor.java

```diff
@@ -298,16 +285,16 @@ private void execute(final EventualMessage msg,
         TracingActor.handleBreakpointsAndStepping(msg, dbg, actor);
       }
 
-      try {
-        if (VmSettings.ACTOR_TRACING) {
-          ActorExecutionTrace.scopeStart(DynamicScopeType.TURN, msg.getMessageId(),
-              msg.getTargetSourceSection());
-        }
-        msg.execute();
-      } finally {
```

Comment:
Is the removal of this finally intentional? Seems to be the safer to use a finally here.